packit_service/service/api/tasks.py

```diff
@@ -67,6 +68,14 @@ def get(self):
                     data["result"]["event"] = Event.ts2str(event)
                 tasks.append(data)
 
+        for task in TaskResult.get_all():
+            data = task.to_dict()
```

Comment:
wow, I didn't know that SA has such method, neat!

Reply:
actually I think it is a method from celery task model